components/NoteEditor.tsx

```
@@ -190,12 +201,19 @@ const NoteEditor: React.FC<NoteEditorProps> = ({ note, onUpdateNote, onDeleteNot
-      />
+      {viewRaw ? (
+        <textarea
+          value={content}
+          onChange={handleContentChange}
+          className="flex-grow w-full p-6 bg-transparent text-text focus:outline-none resize-none text-body leading-relaxed placeholder-text-secondary"
+          placeholder="Start writing your note here, or try dictating!"
+        />
+      ) : (
+        <div
+          className="markdown-preview flex-grow w-full p-6 overflow-y-auto text-text"
+          dangerouslySetInnerHTML={{ __html: previewHtml }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)